thold_functions.php

```diff
@@ -2388,157 +2416,167 @@ function thold_check_threshold(&$thold_data) {
 			}
 
 			if ($notify) {
-				$trigger = false;
+				$etrigger = false;
 			} else {
```

Comment:
Please define the difference between trigger and etrigger?

Reply:
$trigger is a count used elsewhere and was getting overwritten.  The above is only for the get_email_subject function after the above which expects the value to be boolean.